extensions/wikia/WikiaPhotoGallery/WikiaPhotoGallery.class.php

```diff
@@ -402,7 +418,12 @@ public function parse(&$parser = null) {
 				} else if (substr($part, 0, 10) == 'shorttext=') {
 					$shorttext = substr($part, 10);
 				} else {
-					$captionParts[] = trim($part);
+					$tempPart = trim($part);
+
+					//If it looks like Gallery param don't treat it as a caption part
+					if (!in_array($tempPart, $this->mAvalaibleUniqueParams)) {
```

Comment:
Does it mean that we won't display captions like _small_, _landscape_ etc.? Is it working the same way in MediaWiki?